app/services/scrape_onet_codes.rb

```diff
@@ -13,6 +13,6 @@ def call
   end
 
   def format_code(code)
-    code.sub(/^(\d{2})\./, '\1-')
+    code.strip.sub(/^(\d{2})\./, '\1-')
```

Comment:
Strip only tackles beginning and end. To remove all white space you can use `squish`